### ISSUE TITLE
fix:FluentTextArea enter key valid EditForm

### ISSUE THREAD

**Issue by be-swarm** [microsoft/fluentui-blazor#1085]
# 🐛 Bug Report
When FluentTextArea is used in EditForm when enter key is pressed EditForm is submitted

## 💻  Code Sample
````
<EditForm Model="@_current" OnValidSubmit="@ActionNewMailTemplate">
    <BeSwarm.Validator.BlazorValidator ValidateContext="_validatorContext" />
    <FluentTextField Label="Code" @bind-Value=_current.TemplateName></FluentTextField>
    <ValidationMessage For="@(() => _current.TemplateName)" />
    <FluentSwitch @bind-Value=_current.IsBodyHtml>Html</FluentSwitch>
    <FluentTextField Label="Sujet" @bind-Value=_current.Subject></FluentTextField>
    <ValidationMessage For="@(() => _current.Subject)" />
    <FluentTextArea @bind-Value=_current.Body Placeholder="message ..."></FluentTextArea>
    <ValidationMessage For="@(() => _current.Body)" />
    <div>
        <FluentButton Type="ButtonType.Submit" Appearance="Appearance.Accent "> Ajouter</FluentButton>
    </div>
</EditForm>
@code
{
   private async Task ActionNewMailTemplate()
   {
      ....
   }
}

````
ActionNewMailTemplate is called when enter key is pressed in FluentTextField 

Is it possible to prevent this ?

**Comment by vnbaaij** [comment]
Yes, remove the `Type=ButtonType.Submit` part from the `FluentButton`.

This is a workaround. We are looking for a bettter solution.

**Comment by pingu2k4** [comment]
Encountering this issue also - Removing the Submit button isn't great as we rely on that for its intended use. As in, we would lose other functionality we intend to keep.

**Comment by vnbaaij** [comment]
See my additional comment in #1050. This is being caused by a bug in the underlying FAST script and can't be fixed until it is fixed there,

**Comment by yamanaiyuki** [comment]
Hello, I had a problem too.
My problem was with editing the bookmark details, not the email body.
I have prepared a way to enable validation for EditForm, without using Type=ButtonType.Submit as mentioned by vnbaaij.

1. Switch from DataAnnotationsValidator to FluentValidationValidator.
2. Use EditContext instead of Model (using both at the same time occurred an error.)
3. Remove Type="ButtonType.Submit" and add OnClick="HandleSubmit".
4. Finally, remove OnValidSubmit.

A working page and a example source are shown below.
https://wonderful-field-057679e00.5.azurestaticapps.net
https://github.com/yamanaiyuki/FluentTextArea1085/blob/main/Pages/WithoutSubmit.razor

In any case, this avoids the crisis of not being able to hit Enter in the TextArea.
Hopefully the underlying problem will be fixed soon.

**Comment by hkusulja** [comment]
Any info or update on this?

**Comment by vnbaaij** [comment]
It is an issue with the v2 web component. As those are no longer being maintained, we need to wait for our v5 (which uses v3 web components)

**Comment by hkusulja** [comment]
so, just to confirm, v5 is not yet out, and this existing v3 is not supported anymore.
so this can be closed?
is this resolved in v5?

**Comment by vnbaaij** [comment]
Sorry, I meant issue is in **v2** WC.

 **CONFIRMED** (in our v5)  this is resolved in **v3** of the WCs (both are not out yet)

**Comment by hkusulja** [comment]
ok, but why then close this as resolved, when is not (visible) ?

**Comment by vnbaaij** [comment]
It is resolved in the next version (hence the v5 label). There is only so much qualifications /labeling we can do.
